spec/controllers/api/v0/order_cycles_controller_spec.rb

```diff
@@ -68,7 +68,7 @@ module Api
                              q: { with_properties: [property1.id, property2.id] }
 
           expect(response.status).to eq 200
-          expect(product_ids).to include product1.id, product2.id
+          expect(product_ids).to eq [product1.id, product2.id]
```

Comment:
Both products have only one property. If you assign property2 to product1 as well then you are testing for duplication. I did that locally and it works. It may be worth adding though, to avoid regressions in the future.

Reply:
Right. I'v added `property2` to `product1` !